src/odemis/acq/stream/_static.py

```diff
@@ -1072,7 +874,7 @@ def _updateCalibratedData(self, bckg=None, coef=None):
 
         # will raise an exception if incompatible
         calibrated = calibration.compensate_spectrum_efficiency(data, bckg, coef)
```

Comment:
The background subtraction need to be updated to support also the T dimension. It's not a big change, but it should just do the same with the T: for a data of shape CT1YX, the background of shape CT111 should be subtracted for each pixel (ZYX) independently), in `compensate_spectrum_efficiency()`
Moreover,  `get_spectrum_data()` must look also for data of shape CT111 (instead of only C1111).